src/test/configuration/index.js

```diff
@@ -3,6 +3,12 @@ const axios = require('axios');
 const { startTestServer, isStageTesting, port } = require('../../server.js');
 axios.defaults.baseURL = `http://localhost:${port}`;
 
+// Check if user is dev before running test
```

Comment:
So glad you commented. Full stop at the end of comments plssss

Reply:
Cool